src/components/Filter.tsx

```diff
@@ -36,6 +36,12 @@ export interface FilterProps {
     /** The text filter. */
     filter?: string;
     /** The function fired when the text filter changes. */
+    onFilterChange?: FilterOnChange;
+    /**
+     * @deprecated
```

Comment:
Thanks for making this a non-breaking change! I like it 👍

Reply:
done! I realized there was another usage a few lines below, updated it as well